packages/sku/src/program/commands/build-ssr/build-ssr.action.ts

```diff
@@ -36,6 +37,9 @@ export const buildSsrAction = async ({
   const { port, cspEnabled } = skuContext;
   await configureProject(skuContext);
   validatePeerDeps(skuContext);
+
+  // Check for unnecessary polyfills and display warnings
+  displayPolyfillWarnings(skuContext.polyfills);
```

Comment:
This comment seems redundant. Maybe renaming the function to `validatePolyfills` to match `validatePeerDeps` (which also scans and warns about stuff) would be clearer?